src/frontend/src/observer/observer_manager.rs

```diff
@@ -251,7 +251,7 @@ impl FrontendObserverNode {
                                     // FIXME: the frontend node delete its fragment for the update
                                     // operation by itself.
                                     self.worker_node_manager
-                                        .remove_streaming_fragment_mapping(&old_fragment_id);
```

Comment:
@chenzl25 can you PTAL? Seems like the FIXME is added by you. I encountered the error here: https://buildkite.com/risingwavelabs/main-cron/builds/2124#018e64df-940f-40af-9e6e-117ca0ca13eb

Reply:
Oops perhaps my git blame is wrong? Github shows it's @yezizp2012 instead..
![Uploading Screenshot 2024-03-22 at 5.08.43 PM.png…]()

Reply:
This `FIXME` is used to remind us that table catalog update and its corresponding fragment update couldn't be done atomically. See `handle_fragment_mapping_notification` for more details.

Reply:
IIRC, this case could be triggered by `Alter  table add column`

Reply:
It was triggered by create sink into table, and the first run was aborted and cleaned when meta was restarted. The panic could be related to the notifications during cleaning and re-creating.